php-frontend/src/main/java/org/sonar/php/cfg/PhpCfgBlock.java

```diff
@@ -36,36 +37,37 @@
 
   private LinkedList<Tree> elements = new LinkedList<>();
 
-  PhpCfgBlock(PhpCfgBlock successor) {
+  PhpCfgBlock(@Nullable PhpCfgBlock successor) {
```

Comment:
I don't like this change.. while I don't know what should be preferred way
(IMHO I would just won't fix issue inside precondition)